index.js

```diff
@@ -1,5 +1,28 @@
 let bookList = [];
```

Comment:
- [ ] We are allowed to have arrays and objects, but they should all be in a class. Consider refactoring this into the Book class as a static property of the class, or go the route of two classes, one for booksCollection and the other defining a Book Object.